### ISSUE TITLE
Datapack doesn't load on server

### ISSUE THREAD

**Issue by pjc5712** [TeamTernate/infinite-music-discs#125]
Hello,

I put the datapack on my server, and it gave the error message:
Failed to load function yumebako_dp:help
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Whilst parsing command on line 1: Invalid chat component: Not a boolean: "true" at position 83: ...d":"true"}<--[HERE]
It's a fabric server running Terralith and a proximity chat mod. I don't think that would cause a problem though.
Let me know if you have a solution, in the meantime I'm going to try deleting various functions in the datapack folder until it works.

Also, thanks for making an amazing datapack!

**Comment by bradytheinventor** [comment]
Yeah Minecraft changed how it parses JSON in 1.19.4 which broke the help function. It just needs a small formatting update to work again, which will be included in the next release. That help function is not essential for the discs to function, you're safe to delete it.

I'm surprised the help function not loading prevented the rest of the datapack from working. If this happens again, you can also try running `/function yumebako_dp:setup_load` instead of deleting the help function.
